.nvmrc

```diff
@@ -1 +1 @@
-14
+14.15.1
```

Comment:
@bbangert  Just a thought that whenever we run `nvm install --lts` it returns the same version. So is there a need to use this file. We can just update the readme.

Reply:
I mentioned this in a different PR, but the reason for a `.nvmrc` file is to provide an additional sanity check that automated tooling can use. For example, I use a shell hook that when I change into a directory, if there's a `.nvmrc` file, it uses `nvm` to switch my active Node. I work on multiple node projects, and using the right version is important, its easy to use the wrong version without this check when switching between projects.

Using just `14` did not catch the difference, so I was confused when I ran `yarn start` today, and it did not start up, because I was on `14.15.0`.

Reply:
https://stackoverflow.com/questions/23556330/run-nvm-use-automatically-every-time-theres-a-nvmrc-file-on-the-directory has some information on how to setup a shell to do this automatically.

Reply:
@bbangert  Actually I totally agree with this point of yours but as per my observations while setting up the project for a newbie the main `README`  file lacks a lot and while doing `npm start` it only gives the errors.There is a great need to enhance the documentation for the complete repo.I think we should merge this PR since the change is necessary.